GuysNight.LethalCompanyMod.BalancedItems/Utilities/ConfigUtilities.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using GuysNight.LethalCompanyMod.BalancedItems.Models;
+
+namespace GuysNight.LethalCompanyMod.BalancedItems.Utilities {
+	public static class ConfigUtilities {
+		public static void SyncConfigForItemOverrides(Item item, out OverrideProperties overrideProperties) {
+			var itemWeight = item.weight;
+			var itemMinValue = item.minValue;
+			var itemMaxValue = item.maxValue;
+			var itemAverageValue = (ushort)Math.Round(new[] { itemMinValue, itemMaxValue }.Average(), MidpointRounding.AwayFromZero);
+
+			//if the overrides container does not contain an override for the current scrap item, add an entry
```

Comment:
Comment should not say "scrap item" as this could be equipment too. Let's generalize this.